pkg/visor/api.go

```diff
@@ -303,6 +305,72 @@ func (v *Visor) SetAppPassword(appName, password string) error {
 	return nil
 }
 
+// SetAppKillswitch implements API.
+func (v *Visor) SetAppKillswitch(appName string, killswitch bool) error {
+	if appName != skyenv.VPNClientName {
+		return fmt.Errorf("app %s is not allowed to set killswitch", appName)
+	}
+
+	v.log.Infof("Setting %s killswitch to %q", appName, killswitch)
+
+	const (
+		killSwitchArg = "-killswitch"
```

Comment:
Are you this is correct format? @Darkren wrote in chat that I need to use it like this `--killswitch=true`

Reply:
The vpn-client/vpn server work with either version: single or double dashed.

The reason I used single dash is for consistency with other arguments like passcode that are also declared with a single dash in the code around there.

I did some reading, and it seems that it actually shouldn't work. Cobra maintains a fork of flag package, that handles single dash [differently](https://github.com/spf13/pflag#command-line-flag-syntax)

However, when I build vpn-client and vpn-server, they accept both single and double dash flags, like [stdlib flag](https://golang.org/pkg/flag/#hdr-Command_line_flag_syntax).
The help message in vpn-server/vpn client actually suggests using single dash flags.

Maybe cobra is configured somehow to use stdlib behavior of flags, I'm not sure. Maybe @Darkren can comment on that.

Reply:
@i-hate-nicknames  is actually right. I tried it the first time with `-killswitch` and it didn't work for me, probably I just messed something up back then. just tried it again, it works